### ISSUE TITLE
Parser bug

### ISSUE THREAD

**Issue by laurentbartholdi** [gap-system/gap#1878]
With the latest git head, I try:

```
gap> a := false;;
gap> a{{1..1}};
```

and lots of empty lines follow:
```
                                                                                                  \
                                                                                                   \
                                                                                                   \
                                                                                                   \
                                                                                                   \
                                                                                                   \
                                                                                                   \
                                                                                                   \
                                                                                                   \
                                                                                                   \
                                                                                                   \
                                                                                                   \
                                                                                                   \
                                                                                                   \
                                                                                                   \
                                                                                                   \
                                                                                                   \
```

**Comment by fingolfin** [comment]
For reference, this is what happens in previous GAP versions:
```
gap> a{{1..1}};
Syntax error: literal expected
a{{1..1}};
  ^
gap> 
```

**Comment by fingolfin** [comment]
With a `git bisect`, I determined that this was introduced by @ChrisJefferson's multi-argument lambda code in ba5ca7482101c91fb7253ac6179dcd8c56cb1d20.

**Comment by ChrisJefferson** [comment]
Found the bug, submitted a fix ( #1882 )

**Comment by ChrisJefferson** [comment]
This is now fixed. The message could perhaps still be better, but there are no silly blank lines.
